libvmaf/include/libvmaf/libvmaf.h

```diff
@@ -30,6 +30,8 @@
 extern "C" {
 #endif
 
+#define MAX_VMAF_THREADS  8 // default maximum threads
```

Comment:
I don't think we want this is the public header.

Reply:
MAX_VMAF_THREADS  has been deleted and framesync has been modified to allocate memory dynamically.